lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -153,10 +153,13 @@ private boolean hasPassedWithOneIncorrectAnswer(KBVItem kbvItem) {
                 && kbvItem.getQuestionAnswerResultSummary().getAnsweredCorrect() == 3;
     }
 
-    private boolean hasMultipleIncorrectAnswers(KBVItem kbvItem) {
+    private boolean hasTooManyIncorrectAnswers(KBVItem kbvItem) {
         return VC_THIRD_PARTY_KBV_CHECK_NOT_AUTHENTICATED.equalsIgnoreCase(kbvItem.getStatus())
                 && Objects.nonNull(kbvItem.getQuestionAnswerResultSummary())
-                && kbvItem.getQuestionAnswerResultSummary().getAnsweredIncorrect() > 1;
+                && (kbvItem.getQuestionAnswerResultSummary().getAnsweredIncorrect() > 1
+                        || (kbvItem.getQuestionAnswerResultSummary().getAnsweredIncorrect() > 0
+                                && kbvItem.getQuestionAnswerResultSummary().getQuestionsAsked()
+                                        <= 3));
```

Comment:
This I think needs to be ==3, with don't want contra-indicator thing file situations i.e < than the 3 out 4 scenario

Reply:
Nice, please could you also update https://github.com/alphagov/di-ipv-cri-kbv-api/blob/main/lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialServiceTest.java#L89 
1st to 
  from      `"Unable to Authenticate,3,1,2,0,",` to Not Authenticated  `"Not Authenticated,3,1,2,0,",`

run the test ( am expecting this to fail)

then update to `"Not Authenticated,3,1,2,0,V03",` this I expect would pass

Reply:
I've tested it and the changes have the effect that you predicted - I've updated the test to have the V03